Scripts/Mac/BuildInstaller.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
+  CONFIGURATION=Debug
+fi
+
+PACKAGEVERSION=$2
+if [ -z $PACKAGEVERSION ]; then
+  PACKAGEVERSION="0.0.0.1"
+fi
+
+INSTALLROOT=$VFS_OUTPUTDIR/"InstallerRoot"
```

Comment:
Variable naming nit: This is not the install root, it's the build output root for this installer project. This name confused me at first because I thought it represented where the bits will be installed, but it's just the staging area for creating the installer.

Reply:
Renamed to Staging.